lib/house.rb

```diff
@@ -0,0 +1,75 @@
+class House
+  attr_reader :string_price,
+              :address,
+              :rooms
+  def initialize(string_price, address)
+    @string_price = string_price
+    @address = address
+    @rooms = []
+  end
+
+  def rooms_sorted_by_category
```

Comment:
This works, but the method we were looking for was `rooms_by_category`.  Keep a close eye on the interaction pattern!